tests/glsl-intrinsic/atomic/atomicStorageBuffer.slang

```diff
@@ -1,57 +1,57 @@
 //TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl -DTARGET_GLSL
 //TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv -emit-spirv-directly -DTARGET_SPIRV
-
-// currently tests do not handle atomic_uint well
 //TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl
 //TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl -emit-spirv-directly 
 #version 430
 
-
-//#define TEST_when_shader_atomic_float2_is_available
+// float2 is currently a very new extension; most hardware lacks 
+// this extension and will fail the test if attempting to use atomic_float2 
+// operations
```

Comment:
Which hardware would it work with?
Isn't Ampere (RTX 30XX) new enough?

Reply:
https://vulkan.gpuinfo.org/displayreport.php?id=28714#device
likely just not added

Reply:
The spec might be newer than hardware.
But it doesn't mean the older hardware wouldn't support.

Reply:
I see that the link shows which extensions are supported for 4090.
And I don't see "VK_EXT_shader_atomic_float" in it.

Reply:
![image](https://github.com/jkwak-work/slang/assets/159081215/5c58b16c-edca-4828-8929-235dce9d9cb5)

Reply:
(missing atomic_float2)